readme.md

```diff
@@ -48,6 +48,37 @@ Closes the server.
 
 - callback: passed along to the existing `server.close` function to auto-register a 'close' event
 
+## Manual operations on counters
+
+It's also possible to manually increment and decrement a counter for a specific socket.
+This can be helpful when you want to lock the server while doing some jobs.
+For example, you can increment a counter when you receive a request via Websocket
+and decrement it once you have processed it.
+
+**stoppable.increment**
+
+```js
+server.stoppable.increment(socket)
+```
+
+Increment a counter for a specified socket. This method returns a new counter.
```

Comment:
Perhaps rephrase to "This method returns the new count." or something like that? Apparently I guessed right, but it really was a guess :)